orchagent/orchdaemon.cpp

```diff
@@ -264,7 +264,11 @@ bool OrchDaemon::init()
 
     if (WarmStart::isWarmStart())
     {
-        warmRestoreAndSyncUp();
+        bool suc = warmRestoreAndSyncUp();
+        if (!suc)
+        {
+            return false;
+        }
```

Comment:
from line 265 - 274 it could be ine one line `return !WarmStart::isWarmStart || warmRestoreAndSyncUp();`

Reply:
It is equivalent. I prefer no change for easy reading.